themes/doks/layouts/_default/_markup/render-heading.html

```diff
@@ -0,0 +1 @@
+<h{{ .Level }} id="{{ .Anchor | safeURL }}" class="link-heading">{{ .Text | safeHTML }} <a class="link-heading-icon" href="#{{ .Anchor | safeURL }}">🔗</a></h{{ .Level }}>
```

Comment:
Is it possible to break this across 3 lines to indent the anchor inside the header and make it easier to read?